static/bidder-info/ads_interactive.yaml

```diff
@@ -0,0 +1,22 @@
+endpoint: "https://bntb.adsinteractive.com/"
```

Comment:
![image](https://github.com/user-attachments/assets/327de4a4-89d2-4dee-89a2-2266ad191324)
not working